src/backend/distributed/master/master_delete_protocol.c

```diff
@@ -268,8 +268,13 @@ master_drop_sequences(PG_FUNCTION_ARGS)
 		appendStringInfo(dropSeqCommand, " %s", TextDatumGetCString(sequenceText));
 	}
 
-	SendCommandToWorkers(ALL_WORKERS, DISABLE_DDL_PROPAGATION);
-	SendCommandToWorkers(ALL_WORKERS, dropSeqCommand->data);
+	if (dropSeqCommand->len != 0)
+	{
+		appendStringInfoString(dropSeqCommand, " CASCADE");
```

Comment:
Basically, we send the `DROP SEQUENCE` command to all workers but `DROP TABLE` command to metadata workers. If a `stop_metadata_sync_to_node` is followed by a `DROP TABLE`, then the distributed table won't be dropped and when it comes to `DROP SEQUENCE`, the command will error out becuase there exists a table depending on the sequence.